backend/src/Manager/StoreOwnerBranch/StoreOwnerBranchManager.php

```diff
@@ -51,6 +51,24 @@ public function createBranch(StoreOwnerBranchCreateRequest $request):?StoreOwner
         return $entity;
     }
 
+    /**
+     * @param StoreOwnerBranchCreateRequest $request
```

Comment:
FYI there is no need to add @param as long as you define it in function 
`createBranchByStoreOwner(StoreOwnerBranchCreateRequest $request)`